docs/variants/protectli_vp2420/test-matrix.md

```diff
@@ -0,0 +1,87 @@
+# Test matrix
+
+## About
+
+The test matrix is used to determine the scope of tests which the DUT is
+subjected from before the release of the new binary. The test scope is the same
+for all platforms of the VP2420 family.
+
+## Module: Dasharo compatibility
+
+| No.  | Supported test suite                              | Test suite ID | Supported test cases                 |
+|:----:|:--------------------------------------------------|:-------------:|:-------------------------------------|
+| 1.   | [Memory HCL][HCL]                                 | HCL           | All                                  |
+| 2.   | [UEFI compatible interface][EFI]                  | EFI           | All                                  |
+| 3.   | [Display ports][DSP]                              | DSP           | DSP002.001, DSP002.002, DSP002.003, DSP003.001, DSP003.002, DSP003.003 |
+| 4.   | [Network boot utilities][NBT]                     | NBT           | All                                  |
+| 5.   | [NVMe support][NVM]                               | NVM           | All                                  |
+| 6.   | [Custom logo][CLG]                                | CLG           | All                                  |
+| 7.   | [Custom boot menu key][CBK]                       | CBK           | All                                  |
+| 8.   | [USB HID and MSC Support][USB]                    | USB           | All                                  |
+| 9.   | [FreeBSD support][BSD]                            | BSD           | All                                  |
+| 10.  | [Debian Stable and Ubuntu LTS support][LBT]       | LBT           | LBT003.001, LBT003.002, LBT004.001, LBT004.002|
+| 11.  | [USB-C/Thunderbolt][UTC]                          | UTC           | UTC004.001, UTC004.001               |
+| 12.  | [M.2 WiFi/Bluetooth][WLE]                         | WLE           | ALL                                  |
+| 13.  | [eMMC support][MMC]                               | MMC           | MMC001.001                           |
+| 14.  | [SMBIOS][DMI]                                     | DMI           | DMI002.001, DMI003.001, DMI004.001, DMI005.001, DMI006.001 |
+| 15.  | [Custom network boot entries][CNB]                | CNB           | CNB001.002                           |
+| 16.  | [M.2 automatic SATA/NVMe switching support][MSS]  | MSS           | MSS001.001                           |
+| 17.  | [Windows booting][WBT]                            | WBT           | WBT001.001                           |
+| 18.  | [Audio subsystem][AUD]                            | AUD           | AUD001.001, AUD001.002, AUD002.001, AUD002.002, AUD003.001, AUD003.002, AUD004.001, AUD004.002, AUD005.001, AUD005.002, AUD006.001, AUD006.002 |
```

Comment:
Are we sure, that the platform is equipped with a micro jack slot?

Reply:
It does not, maybe some tests would work with HDMI/DP, but i don't think there is a need for that so i removed this test.